src/lib/Interfaces.ts

```diff
@@ -126,6 +126,7 @@ export interface ISplit {
   percentage: number;
   unique_id: string | null;
   wallet_address: string;
+  verified_identity: boolean | null;
```

Comment:
@kim-formless How do we do this on profiles today with the verified badge? I was thinking we could use the same schema.

Reply:
@un7c0rn I was looking into using the Profile Manager but it would require making an RPC call for each split owner within the map, versus reading from the splits table that is already fetched. WDYT? 

https://github.com/formless-eng/share/blob/4324fcb96b5d67205a54e4a38bc3f28599785144/frontend/src/managers/ProfileManager.ts#L17

Reply:
@kim-formless Good call.